express/code/blocks/simplified-pricing-cards-v2/simplified-pricing-cards-v2.css

```diff
@@ -31,29 +31,46 @@
   --icon-size: 18px;
   --tooltip-icon-size: var(--spacing-400);
   --premium-icon-size: var(--body-font-size-xs);
-  
+
   /* === TRANSITION VARIABLES === */
   --transition-duration: 0.3s;
   --transition-easing: ease-out;
-  
+
   /* === SPACING VARIABLES (LOCAL) === */
-  --breakpoint-tablet: 600px;    /* Tablet breakpoint */
-  --breakpoint-desktop: 1200px;  /* Desktop breakpoint */
-  --spacing-025: 2px;   /* Small margins and offsets */
-  --spacing-050: 4px;   /* Eyebrow padding, outline offset */
-  --spacing-075: 6px;   /* Small gaps */
-  --spacing-125: 10px;  /* Medium gaps */
-  --spacing-150: 12px;  /* Button padding, border radius */
-  --spacing-475: 38px;  /* Specific component heights */
-  --spacing-550: 44px;  /* Icon positioning */
-  --spacing-1125: 91px; /* Tooltip max-width */
-  --spacing-negative-100: -2px;  /* Small negative offsets */
-  --spacing-negative-200: -4px;  /* Medium negative offsets */
-  --spacing-negative-225: -6px;  /* Specific negative offsets */
-  --spacing-negative-200-alt: -8px; /* Alternative negative offset */
-  --spacing-negative-500: -40px; /* Large negative offsets */
-  --spacing-negative-1100: -37.5px; /* Large negative positioning */
-  --spacing-negative-2250: -9999px; /* Screen reader hide */
+  --breakpoint-tablet: 600px;
+  /* Tablet breakpoint */
+  --breakpoint-desktop: 1200px;
+  /* Desktop breakpoint */
+  --spacing-025: 2px;
+  /* Small margins and offsets */
+  --spacing-050: 4px;
+  /* Eyebrow padding, outline offset */
+  --spacing-075: 6px;
+  /* Small gaps */
+  --spacing-125: 10px;
+  /* Medium gaps */
+  --spacing-150: 12px;
+  /* Button padding, border radius */
+  --spacing-475: 38px;
+  /* Specific component heights */
+  --spacing-550: 44px;
+  /* Icon positioning */
+  --spacing-1125: 91px;
```

Comment:
This seems very specific. And the negative values below also seem to be prone to future break.

is there any other way to accomplish using positive values?